### PR TITLE
Added increment/decrement support to Gauges

### PR DESCRIPTION
**Pull request by GentlemanJ**
@juliusv 

Addresses Issue #64

### CODE REVIEW

File: exporter.go [hunk 1]
```diff
@@ -287,24 +303,27 @@ type StatsDListener struct {
...
 }
 
-func buildEvent(statType, metric string, value float64, labels map[string]string) (Event, error) {
+func buildEvent(statType, metric string, value float64, valueStr string, labels map[string]string) (Event, error) {
 	switch statType {
 	case "c":
 		return &CounterEvent{
 			metricName: metric,
 			value:      float64(value),
+			valueStr:   valueStr,
```
Comment (juliusv) [inline exporter.go]: Do we need to store the `valueStr` in all the metric types, or would it be sufficient to just add a `relative` boolean flag only to the `GaugeEvent` type and then just do this:

```go
if event.relative {
    gauge.Add(event.Value())
} else {
  gauge.Set(event.Value())
}
```

Comment (GentlemanJ) [inline exporter.go]: Yup, that sounds like a much better implementation. I'll do that

**Comment by juliusv** [comment]
@SuperQ It's common practice in Go to omit a field in a struct initialization if you are just going to set it to its zero value (`false`), so I think it's fine as it is.

**Comment by SuperQ** [comment]
Ahh, ok, makes sense.

**Pull request merged**